Sources/AsyncLocationKit/AsyncLocationManager.swift

```diff
@@ -35,28 +35,20 @@ public final class AsyncLocationManager {
     private var locationManager: CLLocationManager
     private var proxyDelegate: AsyncDelegateProxyInterface
     private var locationDelegate: CLLocationManagerDelegate
-    private var desiredAccuracy: LocationAccuracy = .bestAccuracy
     
-    public init() {
-        locationManager = CLLocationManager()
-        proxyDelegate = AsyncDelegateProxy()
-        locationDelegate = LocationDelegate(delegateProxy: proxyDelegate)
-        locationManager.delegate = locationDelegate
-        locationManager.desiredAccuracy = desiredAccuracy.convertingAccuracy
+    public convenience init(desiredAccuracy: LocationAccuracy = .bestAccuracy, allowsBackgroundLocationUpdates: Bool = false) {
+        self.init(locationManager: CLLocationManager(), desiredAccuracy: desiredAccuracy)
     }
```

Comment:
Allows background location updates unused in this initializer

Reply:
Opps! Thanks for the fix and merging my PRs in.

Reply:
Thank you for your PRs)